raisinbread/instruments/instrument_sql/bmi.sql

```diff
@@ -1,15 +1,12 @@
 CREATE TABLE `bmi` (
 `CommentID` varchar(255) NOT NULL default '',
-
-                            `UserID` varchar(255) default NULL,
-
-                            `Examiner` varchar(255) default NULL,
-
-                            `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+`UserID` varchar(255) default NULL,
+`Examiner` varchar(255) default NULL,
+`Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
 `Date_taken` date default NULL,
 `Candidate_Age` varchar(255) default NULL,
 `Window_Difference` varchar(255) default NULL,
-`unit_classification` enum('metric','standard') default NULL,
+`unit_classification` enum('metric','imperial','standard','not_answered') default NULL,
```

Comment:
i think the options should just be metric and imperial so its just the 'standard' in the SQL that needs to be changed to 'imperial'